<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:
qlineargradient(spread:pad, x1:0.502, y1:0.0611111, x2:0.527155, y2:1, stop:0 rgba(0, 255, 104, 255), stop:0.971014 rgba(123, 255, 165, 255))
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="namelineEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordlineEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>  密码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="usernamelineEdit">
       <property name="font">
        <font>
         <family>思源黑体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>返回登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
